src/core/00_github_runner.tf

```diff
@@ -30,3 +30,96 @@ module "github_runner" {
 
   tags = var.tags
 }
+
+locals {
+  repo_owner = "PagoPA"
+  repo_name  = "io-infra"
+  image_name = "ghcr.io/pagopa/github-self-hosted-runner-azure:beta-runner-v2"
```

Comment:
reminder: when we'll apply this PR I want to se if changing image_name azapi will replace the image value